tests/Tester.py

```diff
@@ -0,0 +1,42 @@
+import csv
+
+from apis import WikipediaApi
+from src import HeuristicTester
+from src.heuristics import Heuristics
+
+test_file_path = "tests.txt"
+results_file_path = "results.tsv"
+
+api = WikipediaApi.WikipediaApi
```

Comment:
Pretty sure you still need to instantiate the api and heuristics here instead of in Search
```suggestion
api = WikipediaApi.WikipediaApi()
heuristics = [Heuristics.BfsHeuristic(), Heuristics.DfsHeuristic()]
```